java/Nikiforov-Oleksandr/src/com/zoolatech/lecture2/tasks/_1/Task1.java

```diff
@@ -0,0 +1,62 @@
+package com.zoolatech.lecture2.tasks._1;
+import java.util.Scanner;
+
+/**
+ * Define a class that represents a calculator. The class should provide methods that
+ * accept another value and perform addition, subtraction, multiplication and division
+ * operations on a value stored in a calculator instance. Divides by a zero should be forbidden and ignored.
+ * The class should also provide a method to get a current value. The class should work with both integer and double numbers
+ * (ignore roundoff errors). Assume all operation results fit into the range of values for a current value type.
+ */
+
+public class Task1 {
+    public static void main(String[] args) {
+        Scanner sc = new Scanner(System.in);
+        System.out.println("Value to calculate: ");
+        float value = sc.nextFloat();
+        System.out.println("Calculator started! Input % to stop program.");
+        calculation(value);
+    }
+
+
+    public static void calculation(float value){
+        Calculcator calculcator = new Calculcator(value);
+        float variable = 0, result =0;
+        end :while (true){
+            Scanner sc = new Scanner(System.in);
+            char symbol = sc.next().charAt(0);
+            switch (symbol){
+                case '+' :
+                    System.out.println("Value to calculate: ");
+                    variable = sc.nextFloat();
+                    result = calculcator.adding(variable);
+                    System.out.println("Result = " + result);
+                    break;
+                case '-' :
+                    System.out.println("Value to calculate: ");
+                    variable = sc.nextFloat();
+                    result = calculcator.substracting(variable);
+                    System.out.println("Result = " + result);
+                    break;
+                case '/':
+                    System.out.println("Value to calculate: ");
+                    variable = sc.nextFloat();
+                    result = calculcator.division(variable);
+                    System.out.println("Result = " + result);
+                    break;
+                case '*':
+                    System.out.println("Value to calculate: ");
+                    variable = sc.nextFloat();
+                    result = calculcator.multiplication(variable);
+                    System.out.println("Result = " + result);
+                    break;
+                case '%':
+                    System.out.println("Bue!");
+                    break end;
```

Comment:
instead of a labeled break you could also use `return` without arguments or the do-while loop with the exit condition `symbol = '%' `

Reply:
One more idea: you can split the accepting input for the operation and checking for the exit symbol into two operations: this would allow to remove the common code from case blocks.
e.g.,
```
variable = sc.nextFloat();
shouldExit = sc.next();
```